easybuild/easyconfigs/a/Alpha/Alpha-20190607-foss-2019b-Python-2.7.16.eb

```diff
@@ -0,0 +1,82 @@
+easyblock = 'Bundle'
+
+name = 'Alpha'
+local_commit = 'e027eb0fea1e'
+version = '20190607'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.lirmm.fr/~swenson/alpha/alpha.htm'
+description = "Alpha is a tool designed for detailed comparative study of bacteriophage genomes."
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '2.7.16'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('Biopython', '1.75', versionsuffix),
+    ('BLAST+', '2.9.0'),
+    ('Clustal-Omega', '1.2.4'),
+    ('GenomeTools', '1.6.1'),
+    ('GTK+', '3.24.13'),
+    ('matplotlib', '2.2.4', versionsuffix),
+    ('networkx', '2.2', versionsuffix),
+    ('pydot', '1.4.1'),
+    ('PyGObject', '3.34.0', versionsuffix),
+    ('pygraphviz', '1.5'),
+    ('python-igraph', '0.8.0'),
+]
+
+components = [
+    ('pyinclude', '0f8a00b8c10c', {
```

Comment:
I understand you need to use the commit ID in `download_filename`, but here you can use something that resembles a version, like a datestamp corresponding to this commit, which is a little bit cleaner?

Reply:
Fixed :+1: